src/plugins/dashboard/public/plugin.tsx

```diff
@@ -341,78 +305,49 @@ export class DashboardPlugin
   }
 
   public start(core: CoreStart, plugins: DashboardStartDependencies): DashboardStart {
-    const { notifications, overlays, application, theme, uiSettings } = core;
-    const { uiActions, data, share, presentationUtil, embeddable } = plugins;
-
-    const dashboardCapabilities: Readonly<DashboardCapabilities> = application.capabilities
-      .dashboard as DashboardCapabilities;
-
-    const SavedObjectFinder = getSavedObjectFinder(core.savedObjects, uiSettings);
-
-    const expandPanelAction = new ExpandPanelAction();
-    uiActions.registerAction(expandPanelAction);
-    uiActions.attachAction(CONTEXT_MENU_TRIGGER, expandPanelAction.id);
+    const { uiSettings } = core;
+    const { uiActions, share, presentationUtil } = plugins;
+    this.startDashboardKibanaServices(core, plugins, this.initializerContext).then(() => {
```

Comment:
This is already so much cleaner now that each action doesn't need a bunch of services passed in - can't wait to remove the last bit :fire: